### PR TITLE
Added branch lesson-3, there added image, made list, table, form.

### PR DESCRIPTION
**Pull request by IrinaBreiksh**
Prettier doesn't work

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,138 @@
...
+          <li>Краткая история Web и как устроен интернет</li>
+          <li>Как устроен браузер и как он получает страницы по сети</li>
+          <li>Что такое Git</li>
+          <li>Создание первого репозитория</li>
+          <li>О файле .gitignore</li>
+        </ul>
+      </li>
+      <li>HTML-вёрстка – первые шаги
+        <ul>
+        <li>Настройка рабочей среды</li>
```
Comment (5tronciy) [inline index.html]: Add padding to nested tags.

File: index.html [hunk 2]
```diff
@@ -1,10 +1,138 @@
...
+        <li>Краткая история HTML</li>
+        <li>Типы тегов и атрибуты</li>
+        <li>Теги для работы с текстом</li>
+        <li>Создание собственной страницы</li>
+        <li>Основы ветвления в Git</li>
+        <li>Создание pull request на GitHub</li>
+      </ul>
```
Comment (5tronciy) [inline index.html:59]: Close tag should be on the same level as open tag.

File: index.html [hunk 3]
```diff
@@ -1,10 +1,138 @@
...
+        <td>&#60tbody&#62</td>
+        <td>Groups the body content in a table</td>
+      </tr>
+      <tr>
+        <td>&#60tfoot&#62</td>
+        <td>Groups the foot content in a table</td>
+      </tr>
+      </tbody>
+<tfoot>
+</tfoot>
```
Comment (5tronciy) [inline index.html]: You may not use the optional tags if there is no content.

File: index.html [hunk 4]
```diff
@@ -1,10 +1,138 @@
...
+    </table>
+    <p></p>
```
Comment (5tronciy) [inline index.html]: Still no reason to do it.

File: index.html [hunk 5]
```diff
@@ -1,10 +1,138 @@
...
+      <fieldset>
+        <legend>My first form</legend>
+        <label for="name">Name</label> <br>
+        <input id="name"  type="text" name="Name" placeholder="Name" required/><br>
+        <label for="email">Email</label> <br>
+        <input id="email"  type="email" name="Email" required/> <br>
+        <label for="password">Password</label> <br>
+        <input id="password"  type="password" name="password" required/> <br>
+        <label for="level">Your programming level</label> <br>
+        <input id="level"  type="range" name="level" min="0" max="10" />
```
Comment (5tronciy) [inline index.html]: "0" doesn't match range "1-10".

File: index.html [hunk 6]
```diff
@@ -1,10 +1,138 @@
...
+        <input id="html-1" type="checkbox" name="html" value="off" />
+        <label for="html-1">Hyperlinks and Text Markup Language</label> <br>
+        <input id="html-2" type="checkbox" name="html" value="off" />
+        <label for="html-2">Home Tool Markup Language</label> <br>
+        <input id="html-3" type="checkbox" name="html" value="on"/>
+        <label for="html-3">Hyper Text Markup Language</label> <br>
+        <p>
+          What does  HTTP stand for&#63
+        </p>
+        <input id="http-1" type="checkbox" name="http" value="off" />
```
Comment (5tronciy) [inline index.html]: Look at the task carefully and change the inputs in this block.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Looks good. 
Few remarks only.

File: index.html (hunk 6 above)
Comment (IrinaBreiksh) [reply index.html]: Changed

File: index.html (hunk 5 above)
Comment (IrinaBreiksh) [reply index.html]: Ok

File: index.html (hunk 4 above)
Comment (IrinaBreiksh) [reply index.html]: Ok

File: index.html (hunk 3 above)
Comment (IrinaBreiksh) [reply index.html]: Ok

File: index.html (hunk 2 above)
Comment (IrinaBreiksh) [reply index.html:59]: Ok, changed

File: index.html (hunk 1 above)
Comment (IrinaBreiksh) [reply index.html]: Ok

File: index.html [hunk 7]
```diff
@@ -1,10 +1,172 @@
...
+          </li>
+          <li>
+            О файле .gitignore
+          </li>
+        </ul>
+      </li>
+      <li>
+        HTML-вёрстка – первые шаги
+        <ul>
+        <li>
```
Comment (5tronciy) [inline index.html:38]: `<li>` is nested in `<ul>` here.

```
<ul>
  <li>...</li>
</ul>
```

File: index.html (hunk 2 above)
Comment (5tronciy) [reply index.html:59]: It is not changed.

File: index.html [hunk 8]
```diff
@@ -15,32 +15,69 @@ <h1>Моя первая страница</h1>
...
       <li>Знакомство с Web
         <ul>
-          <li>Краткая история Web и как устроен интернет</li>
-          <li>Как устроен браузер и как он получает страницы по сети</li>
-          <li>Что такое Git</li>
-          <li>Создание первого репозитория</li>
-          <li>О файле .gitignore</li>
+          <li>
+            Краткая история Web и как устроен интернет
+          </li>
```
Comment (5tronciy) [inline index.html:20]: It is unnecessary changes. 
And as for me `<li>Краткая история Web и как устроен интернет</li>` looks better than

```
<li>
  Краткая история Web и как устроен интернет
</li>
```

**Review by 5tronciy — CHANGES_REQUESTED** [review]
You didn't fix code style issues. I strongly recommend you to use Prettier.

File: index.html (hunk 8 above)
Comment (IrinaBreiksh) [reply index.html:20]: Ok, I agree, changed

**Comment by IrinaBreiksh** [comment]
> You didn't fix code style issues. I strongly recommend you to use Prettier.

It didn't work, I don't know how to fix it. But when I deleted everything, it started to work
